PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -3105,10 +3113,25 @@ int Progress::createProgressWindow()
 
 RECT Progress::adjustSizeAndPos(int width, int height)
 {
-	RECT win, maxWin;
-
+	RECT maxWin;

```

Comment:
While you're at it, maybe it's a good idea to zero init `maxWin`, so we don't read random garbage if `GetWindowRect` fails?